Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN /bin/bash -e /scripts/ubuntu_apt_cleanmode.sh && \
         curl \
     && \
     /bin/bash -e /clean.sh && \
+    # To remove systemd binary
```

Comment:
this is not the degree of comment I was looking for ;)

Instead, something like "out of order execution somehow required, has a dpkg error if performed before the clean script"

Reply:
Apologies @bryanlatten updated the comment to be more meaningful.